### ISSUE TITLE
Wireframe *FIGMA*

### ISSUE THREAD

**Issue by BEATM-P** [omah03/Usable-Machine-Learning-#2]
https://www.figma.com/team_invite/redeem/ZCuNxlY8JGFF66YWTSiyse

**Design a wireframe using FIGMA on Thursday **
use figma for wireframes, prepare desing thursday after 

Target: People with some knowledge, but not specififc to ML
e.g. averge computer science bachelor student

Features:
Training Interface
- start training
- interrupt training
- continue training
- adjust parameters (e.g., learning rate, loss function, momentum, dropout-rate) 
           before the training
           during the training

Training Monitor
- display accuracy and loss over time for training set
- indicate point in training where a parameter was changed

Explainable AI
TBD

Model Creator
- create models using a GUI (2 P)
- change the composition of layers (1 P)
- change aspects of the layers (e.g., sizes) (0.5 P)
